BUILD.bazel

```diff
@@ -512,6 +512,7 @@ exeunitsrc =  [ "rtl/ALU.sv",
 
 [build_openroad(
         name = exe,
+        entrypoint = "//:entrypoint.mk",
```

Comment:
Can this be a default argument inside build_openroad()?

Reply:
Unfortunately this must be specified as argument passed from the context of repo which uses `bazel-orfs`. That is because `entrypoint.mk` must be located in the repo which uses `bazel-orfs`. This is a hack which works around the issues which were encountered when we tried to run bazel rules spawned by `build_openroad()`.

In 2-repository setup (`megaboom` loading `build_openroad` macro definition from `bazel-orfs`) during `bazel build`, a sandbox is created where dependencies from `megaboom` for given bazel rule land in the root directory of the sandbox, while dependencies from `bazel-orfs` (e.g. config.mk and orfs script) land under subdirectory `external/bazel-orfs` in the sandbox.
`Config.mk` lands under `external/bazel-orfs` and `orfs` script was changing directory to this location in order to run make targets. This was failing due to incorrect paths to sources from `megaboom` repository (e.g. SDC file). We were unable to force bazel to provide correct paths to those dependencies because it is not something that bazel was designed to do.
`entrypoint` hack allows us to not cd into `external/bazel-orfs` in `orfs` script because make targets will be defined straight in the root of the sandbox by including `config.mk`. That way all paths to dependencies are specified correctly and the flow works as expected for `L1MetadataArray` example.

Reply:
And that's how we make sausages...